spec/system/service_manual_homepage_spec.rb

```diff
@@ -0,0 +1,69 @@
+RSpec.describe "Service Manual homepage" do
```

Comment:
Only note I have is that in this file a lot of the expectation values are written out explicitly rather than as paths into the example response (eg: https://github.com/alphagov/frontend/pull/4674/commits/5e41596180cacef25003076ed141eafad8e3fcf3). I'm in two minds about this, because I personally prefer things written out explicitly, but I have nothing to back up that preference rather than "I like it", and the idiom of the changes we've made recently has been to have the paths into the response, so this should probably be following that.

Reply:
Thanks @KludgeKML. I think the only test where the text comes from the content item is line 48, so I've updated that one, but let me know if there are others I can change.